website/content/contributors/getting-started.md

```diff
@@ -0,0 +1,105 @@
+
+At the same time, it's important to maintain a level of professional conduct to ensure that contributors don't inadvertently self-sabotage. This includes following the project's code of conduct, adhering to coding standards, and contributing code that is well-tested and documented. ie. The [CNCF Community Code of Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md)
+It's also important to be open to feedback and criticism, and to work collaboratively with other members of the community to improve the project. People of all skill levels work hand-in-hand to improve their projects.
+
+### Choose a Project
+When it comes to contributing to open source, choosing the right project is crucial. A good place to start is the [Cloud Native Computing Foundation (CNCF)](https://www.cncf.io/), which hosts dozens of popular open source projects in the [cloud native ecosystem](https://landscape.cncf.io/). There's a CNCF project for almost every area of technology, from container orchestration to service mesh to storage. For instance, if you're interested in containerization, you might consider contributing to [Kubernetes](https://kubernetes.io/), the de-facto standard for container orchestration. Or, if you're interested in observability and monitoring, you might consider contributing to [Prometheus](https://prometheus.io/), which provides powerful metrics collection and analysis. Other popular CNCF projects include [Envoy](https://www.envoyproxy.io/), [Fluentd](https://www.fluentd.org/), and [Jaeger](https://www.jaegertracing.io/).
+
+When choosing a project to contribute to, it's important to consider your interests and skill level. Look for projects that align with your strengths and that you're passionate about. CNCF projects have a large and supportive community, with many resources available for contributors. It is very important to start by reading the project documentation and exploring the issue tracker. If you're new to the project, look for issues labeled as "good first issue" or "help wanted" - these are often small, well-defined tasks that are a good place to start.
+
+### Understand the Project
+Using an open-source project before becoming a contributor is an essential step to understand the project's functionality and architecture. If available, follow the project’s tutorial or getting started documentation and then try it out yourself to become familiar with how the project works and its capabilities. It helps contributors identify issues and challenges that the project faces, allowing them to get familiar with the codebase and the project's design principles. By testing the project, contributors can provide valuable feedback to the development team, which can help improve the project's quality and usability. Additionally, testing the project can help contributors identify areas where they can contribute to the project, allowing them to make more significant contributions and become more effective contributors.
```

Comment:
Maybe mention roadmap? As in, checking the roadmap of the project can be helpful to understand what the project is and what it will become in the future.

Just an idea...

Reply:
Once we have a workable roadmap, let's revisit and add it to the doc.